### PR TITLE
Part one spruce up

### PR DESCRIPTION
**Pull request by hollisjamison**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,64 +1,131 @@
...
+                <input type="number" pattern="[0-9]*" class="form-control" placeholder="" aria-label="Number One"
+                    aria-describedby="basic-addon1" name="numberOne"></input>
+            </div>
 
+            <!-- Input box two-->
+            <div class="input-group mb-3" style="max-width: 95%;margin:0 auto;">
+                <div class="input-group-prepend">
+                    <span class="input-group-text" id="basic-addon1">Number Two:</span>
+                </div>
+                <input type="number" pattern="[0-9]*" class="form-control" placeholder="" aria-label="Number Two"
```
Comment (john-rice) [inline index.html:92]: Nice use of pattern.

File: index.html [hunk 2]
```diff
@@ -1,64 +1,131 @@
...
+                <div class="input-group-prepend">
+                    <span class="input-group-text" id="basic-addon1" >Result:</span>
+                </div>
+                <input type="text" class="form-control" placeholder="0" aria-label="Result"
+                    aria-describedby="basic-addon1" name="result" disabled></input>
+            </div>
+        </div>
+ </div>
+ </form>
+    <!-- Footer -->
```
Comment (john-rice) [inline index.html:120]: Nice job on the foooter.

File: index.html [hunk 3]
```diff
@@ -1,64 +1,131 @@
...
+                    (-)</button>
+                <button type="button" onclick="calculator.multiplyNumbers()" class="btn btn-primary">Multiply
+                    (x)</button>
+                <button type="button" onclick="calculator.divideNumbers()" class="btn btn-primary">Divide (÷)</button>
+                <button type="button" onclick="window.location.reload()" class="btn btn-danger">Clear
+                    Calculator</button>
+            </div>
+            <!-- Display the result! -->
+        <div class="card-footer">
+            <div class="input-group mb-3" style="max-width: 98%;margin:0 auto;">
```
Comment (john-rice) [inline index.html:110]: Looks great. Not that the style attribute could be removed. The value there could be moved to the style tag with a new css selector or an update to mb-3 or input-group within the style tag in the header.

File: index.html [hunk 4]
```diff
@@ -1,64 +1,131 @@
...
+            
+                <!-- Create the function buttons -->
+                <div class="btn-group" role="Group" style="padding-bottom: 15px;margin:0 auto;">
+                <button type="button" onclick="calculator.addNumbers()" class="btn btn-primary">Add (+)</button>
+                <button type="button" onclick="calculator.subtractNumbers()" class="btn btn-primary">Subtract
+                    (-)</button>
+                <button type="button" onclick="calculator.multiplyNumbers()" class="btn btn-primary">Multiply
+                    (x)</button>
+                <button type="button" onclick="calculator.divideNumbers()" class="btn btn-primary">Divide (÷)</button>
+                <button type="button" onclick="window.location.reload()" class="btn btn-danger">Clear
```
Comment (john-rice) [inline index.html:105]: Nice addition of the clear button.

File: index.html [hunk 5]
```diff
@@ -1,64 +1,131 @@
...
 
+            
+                <!-- Create the function buttons -->
+                <div class="btn-group" role="Group" style="padding-bottom: 15px;margin:0 auto;">
+                <button type="button" onclick="calculator.addNumbers()" class="btn btn-primary">Add (+)</button>
+                <button type="button" onclick="calculator.subtractNumbers()" class="btn btn-primary">Subtract
+                    (-)</button>
+                <button type="button" onclick="calculator.multiplyNumbers()" class="btn btn-primary">Multiply
+                    (x)</button>
+                <button type="button" onclick="calculator.divideNumbers()" class="btn btn-primary">Divide (÷)</button>
```
Comment (john-rice) [inline index.html:104]: Nice use of the division symbol.

File: index.html [hunk 6]
```diff
@@ -1,64 +1,131 @@
...
+            <div class="input-group mb-3" style="max-width: 95%;margin:0 auto;">
+                <div class="input-group-prepend">
+                    <span class="input-group-text" id="basic-addon1">Number Two:</span>
+                </div>
+                <input type="number" pattern="[0-9]*" class="form-control" placeholder="" aria-label="Number Two"
+                    aria-describedby="basic-addon1" name="numberTwo"></input>
+            </div>
 
+            
+                <!-- Create the function buttons -->
```
Comment (john-rice) [inline index.html:97]: inline style is ok here for `style="padding-bottom: 15px;margin:0 auto;"` but could be set in the header style tag instead.

For rough example:
```
<style>
.calc-btn-group {
  padding-bottom: 15px;
  margin:0 auto;
}
</style>
```

And then your tag can be:

```
<div class="btn-group calc-btn-group" role="Group">
```

**Review by johnny-rice — COMMENTED** [review]
Great work.

**Pull request closed without merging**